config_defaults_inc.php

```diff
@@ -4593,6 +4650,7 @@
 	'sponsor_threshold',
 	'sponsorship_currency',
 	'sponsorship_enum_string',
+	'safe_fonts_arr',
```

Comment:
should be inserted before search_title to keep sorting

Reply:
I also suggest dropping the `_arr` suffixes.  I know we may have used them before, but they a lot of our configs support arrays and don't have such suffix.

Reply:
How about `font_choices` and `font_choices_local`?  Such names are better and follow the approach of having a common prefix for related configs.

Reply:
Actually `font_family_choices` and `font_family_choices_local`.